jbehave-support-core/docs/Expression-commands.md

```diff
@@ -185,16 +185,27 @@ This command simply do the sum of parameters.
 > Result of the command: `7`
 
 #### RANDOM_DATE
-Generates random date in range 1970 - 2059.
+Generates random date in range 1970 - 2059 by default.
+Takes up to 2 optional parameters to specify start and/or end date in several formats:
+- YYYY
+- YYYY.M
+- YYYY.M.D
 
 > Example:
 > ```
 > {RANDOM_DATE}
+> {RANDOM_DATE:1999}        //Random date since 1.1.1999 till 31.12.2059
+> {RANDOM_DATE:2000.5}      //Random date since 1.5.2000 till 31.12.2059
```

Comment:
Minor - I would rather use ISO8601 YYYY-MM-DD format in the comments to be clear what is month and what is day